modules/qualitycontrol/CodecSelection.js

```diff
@@ -281,7 +204,43 @@ export class CodecSelection {
             return;
         }
 
-        session.setVideoCodecs(selectedCodecOrder);
+        session.setVideoCodecs(selectedCodecOrder, this.screenshareCodec?.jvb);
+    }
+
+    /**
+     * Changes the codec preference order when initiated by the {@link QualityController} to improve video quality.
```

Comment:
Nitpick: this class doesn't know why this function does, that's the responsibility of the quality controller. So I wouldn't mention the quality aspect of it.